.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: "[Bug]"
-labels: ''
+title: ""
```

Comment:
I am not sure whether we want to remove a title...

Reply:
If there will be an automatic label, there will not be of any use of a prefix in the title saying the same thing. It would be better to prefix with e.g. module name, as was custom in the trac system, but that obviously has to be made manually by the issue reporter.

Reply:
I have no strong opinion, what about original author @neteler ?

Reply:
As written elsewhere, I would miss the "[Bug] ..." in the email inbox. That's why I added it originally.

If ppl would report with module name prefix, sure (I regularly edit PRs for that (so that it starts to make sense in the ChangeLog file).

Of course I can also add some labeling in my inbox if majority here wants no prefix.

Reply:
+1 for keeping `[Bug]/[Feature]` prefix. Empty titles can be fixed manually or by GitHub Action as Vaclav mentioned (let's create new PR for that if needed).

Reply:
To add to my point, currently #590 has no title except for `[Bug]`.

Reply:
Error done by the reporter, isn't it?